buildpipeline/pipelines.json

```diff
@@ -268,6 +293,9 @@
       "BuildParameters": {
         "PB_BuildType": "Release"
       },
+      "ReportingParameters": {
+        "PB_BuildType": "Release"
+      },
```

Comment:
I'm less confident that this will get BuildType showing up in MissionControl, since this ReportingParameter is on the pipeline containing the subjobs, not the subjobs themselves. If this info doesn't get inherited by those jobs, then I think I'd need to refactor this file to look like CoreFX's, unless there's a way for the jobs to do something like:

`"ReportingParameters": {
    "PB_BuildType": "$(PB_BuildType)"
}`